src/main/resources/anniv5Resources/localization/eng/summonspack/Cardstrings.json

```diff
@@ -47,7 +47,7 @@
   },
   "${ModID}:Training": {
     "NAME": "Training",
-    "DESCRIPTION": "At the end of your turn Evoke your next orb !M! times.",
+    "DESCRIPTION": "At the end of your turn Evoke your next Orb !M! times.",
```

Comment:
As part of making this pack I did a sweep through text to make Orb consistently capitalized, which is why there are a bunch of changed JSON files. I also fixed a few punctuation issues along the way.